source/install/transport-encryption/config.rst

```diff
@@ -0,0 +1,22 @@
+=============================================
+Configuring Transport Encryption
+=============================================
+
+The components of the Mattermost setup are shown in the following diagram,
+including the transport encryption used. Aside from the encryption between the
+Mattermost Application Server, all transports rely on TLS encryption.
+
+**Note:** The transport between the Application servers is not used by default and
+requires additional setup steps. Enhancing the core product to include automatic
+encryption between cluster nodes is in progress and planned for the 5.18 release.
```

Comment:
Did this land for 5.18?

Reply:
Thanks, I suspect it will be 5.20 - Edited!